libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/ActionChangeType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// How to modify an action sequence.
+    /// </summary>
+    public enum ActionChangeType
```

Comment:
Naming: The C# team tends to call Enums with plural names, i.e. ActionChangeTypes. Not sure if we want to adhere to that rule or not